beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -723,6 +723,7 @@ impl TestRig {
                     (ev.work_type() == beacon_processor::RPC_BLOBS).then_some(())
                 })
                 .unwrap_or_else(|e| panic!("Expected blobs work event: {e}")),
+            ResponseType::CustodyColumn => todo!(),
```

Comment:
```suggestion
            ResponseType::CustodyColumn => todo!(), //TODO(das)
```

Reply:
Added comment in 96e2363d721f55d79f66f98f15fd61a854c7dc62